packages/site-content/eleventy.config.prod.cjs

```diff
@@ -4,13 +4,17 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+const eleventyNavigationPlugin = require('@11ty/eleventy-navigation');
+
 module.exports = (eleventyConfig) => {
+  eleventyConfig.addPlugin(eleventyNavigationPlugin);
   eleventyConfig.addPassthroughCopy('site/assets');
   eleventyConfig.addPassthroughCopy({'../site-client/bundled': 'js'});
   return {
     dir: {
       input: 'site',
       output: '_site',
     },
+    pathPrefix: '',
```

Comment:
Comment for this? I'm not sure why it's here (and why it's not in the `dev` config)

Reply:
oh, I can remove now. I was trying to get eleventyNavigation working from plain JS instead of Nunjucks and it was looking for the `url` filter. I fixed it by using the plugin API dynamically attached to the eleventyConfig object.